### ISSUE TITLE
Cursor Does Not Re-appear if RPCS3 Crashes

### ISSUE THREAD

**Issue by OnceACodeMonkey** [isJuhn/KAMI#41]
I hid my cursor with KAMI and RPCS3 Crashed. When it did, my cursor remained invisible. It was only visible when I passed it over text fields because it turned into a capital I. I couldn't reveal my cursor with KAMI after RPCS3 had crashed. I had to relaunch RPCS3 and boot the game I was playing again to reverse the process, all while using an invisible cursor.

**Comment by isJuhn** [comment]
You should not use the hide cursor option with RPCS3 unless you must, it is a bad feature and you just experienced why. but if something goes wrong we must be able to still toggle KAMI and reset the cursor, I have added handling such that if you close KAMI while the cursor is hidden it should reset.

**Comment by OnceACodeMonkey** [comment]
Yeah, I also discovered that it stays hidden if KAMI crashes, too. So instead of using KAMI's hide cursor feature, I've been using this AutoHotKey script to hide my cursor: https://www.autohotkey.com/docs/commands/DllCall.htm#HideCursor. No problems so far. Maybe we could use that as a workaround until KAMI's hide cursor feature is more stable.
